libjob_queue/src/ext_job.c

```diff
@@ -808,6 +842,27 @@ void ext_job_fprintf_config(const ext_job_type * ext_job , const char * fmt , FI
   fprintf(stream , fmt , ext_job->name , ext_job->config_file );
 }
 
+config_item_types ext_job_iget_argtype( const ext_job_type * ext_job, int index) {
+  return int_vector_safe_iget( ext_job->arg_types , index );
+}
+
+
+static void ext_job_iset_argtype_string( ext_job_type * ext_job , int iarg , const char * arg_type) {
+  config_item_types type = CONFIG_INVALID;
+
+  if (strcmp( arg_type , JOB_STRING_TYPE) == 0)
```

Comment:
The body of this function could also be moved to the new `job_key_defintions` and then be shared with `workflow_job.c`